docs/user-guides/script-annotations.md

```diff
@@ -41,6 +41,12 @@ The fields allowed in the `Parameter` annotations are: `name`, `enum`, and `desc
 variable name if not provided (when exporting to YAML or viewing in the Argo UI, the `name` variable will be used). A
 `default` must be set using standard Python syntax, i.e. `x: int = 42`.
 
+### Custom Serialisation
+
+If you are using a type that is not a built in (`int`, `str`, `dict` etc) or a Pydantic `BaseModel`, you can provide
+functions to the Parameter's `loader` and `dumper` attributes, which will be used by the Hera Runner (and so requires
+the use of "runner" templates).
```

Comment:
should we add "or if your objects are not serialized as JSON"?

Reply:
Yep good idea, that was actually the original request from #813 🤣